src/topology/continuous_on.lean

```diff
@@ -216,6 +216,31 @@ lemma is_closed.mem_of_nhds_within_ne_bot {s : set α} (hs : is_closed s)
   {x : α} (hx : ne_bot $ nhds_within x s) : x ∈ s :=
 by simpa only [hs.closure_eq] using mem_closure_iff_nhds_within_ne_bot.2 hx
 
+lemma eventually_eq_nhds_within_iff {f g : α → β} {s : set α} {a : α} :
+  (f =ᶠ[nhds_within a s] g) ↔ ∀ᶠ x in 𝓝 a, x ∈ s → f x = g x :=
+mem_inf_principal
+
+lemma eventually_eq_nhds_within_of_eq_on {f g : α → β} {s : set α} {a : α} (h : eq_on f g s) :
+  f =ᶠ[nhds_within a s] g :=
+mem_inf_sets_of_right h
+
+lemma set.eq_on.eventually_eq {f g : α → β} {s : set α} {a : α} (h : eq_on f g s) :
```

Comment:
I know that I suggested this name but now I think that it should be used for `eq_on f g s → f =ᶠ[principal s] g`, so I think that this lemma should be deleted or renamed.